features/apimgt/org.wso2.carbon.apimgt.throttling.siddhi.extension.feature/src/main/resources/conf/eventpublishers/tokenRevokeEventPublisher.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<eventPublisher name="jwtRevocationPublisher" statistics="disable"
```

Comment:
this should be token revocation publisher, i assume.

Reply:
resolved